contracts/BSCValidatorSet.sol

```diff
@@ -331,11 +335,16 @@ contract BSCValidatorSet is IBSCValidatorSet, System, IParamSubscriber, IApplica
         uint256 index = currentValidatorSetMap[valAddr];
 
         if (isSystemRewardIncluded == false) {
-            systemRewardRatio = INIT_SYSTEM_REWARD_RATIO;
+            systemRewardBaseRatio = INIT_SYSTEM_REWARD_RATIO;
             burnRatio = INIT_BURN_RATIO;
             isSystemRewardIncluded = true;
         }
 
+        uint256 systemRewardRatio = systemRewardBaseRatio;
+        if (turnTerm > 1 && systemRewardAntiMEVRatio > 0) {
+            systemRewardRatio += systemRewardAntiMEVRatio * (block.number % turnTerm) / (turnTerm - 1);
```

Comment:
![image](https://github.com/bnb-chain/bsc-genesis-contract/assets/7310198/fb7065f1-b9ae-4f99-b5c0-503d977769a7)
This simulation have an assumption that the block 100, 105 and 110 is the first block of a validator.  However, `block.number % turnTerm ==0 ` does not means the first block of a validator.